main.go

```diff
@@ -47,29 +55,60 @@ func runMain() error {
 		return err
 	}
 
-	camera := lepton3.New(conf.SPISpeed)
-	err = camera.Open()
-	if err != nil {
-		return err
+	var camera *lepton3.Lepton3
+	defer func() {
+		if camera != nil {
+			camera.Close()
+		}
+	}()
+
+	for {
+		camera = lepton3.New(conf.SPISpeed)
+		err = camera.Open()
+		if err != nil {
+			return err
+		}
+		camera.SetLogFunc(func(t string) { log.Printf(t) })
+
+		runRecordings(conf, camera)
+		err := runRecordings(conf, camera)
```

Comment:
You probably don't want to call runRecordings twice in a row :)

Reply:
Whoops...